partitioner/src/main/java/io/confluent/connect/storage/partitioner/PartitionerConfig.java

```diff
@@ -78,10 +82,20 @@
   public static final String TIMEZONE_DEFAULT = "";
   public static final String TIMEZONE_DISPLAY = "Timezone";
 
-  // Schema group
-  public static final String SCHEMA_GENERATOR_CLASS_CONFIG = "schema.generator.class";
-  public static final String SCHEMA_GENERATOR_CLASS_DOC = "The schema generator to use with partitioners.";
-  public static final String SCHEMA_GENERATOR_CLASS_DISPLAY = "Schema Generator Class";
+  public static final String TIMESTAMP_EXTRACTOR_CLASS_CONFIG = "timestamp.extractor";
+  public static final String TIMESTAMP_EXTRACTOR_CLASS_DOC =
+      "The extractor that gets the timestamp for records when partitioning with ``TimeBasedPartitioner``. It can be " +
+          "set to ``Wallclock``, ``Record`` or ``RecordField`` in order to use one of the built-in timestamp " +
+          "extractors or be given the fully-qualified class name of a user-defined class that extends the" +
+          "``TimestampExtractor`` interface.";
+  public static final String TIMESTAMP_EXTRACTOR_CLASS_DEFAULT = "Wallclock";
```

Comment:
Is this ok as a default given https://github.com/confluentinc/kafka-connect-storage-cloud/blob/master/kafka-connect-s3/src/main/java/io/confluent/connect/s3/TopicPartitionWriter.java#L193?